### PR TITLE
feat: tracetest executor

### PR DESCRIPTION
**Pull request by vLia**
## Pull request description 

Updated code from https://github.com/kubeshop/testkube/pull/3691 to include the necessary updates needed for newer Testkube versions

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: contrib/executor/tracetest/pkg/runner/runner.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+	// Get TRACETEST_ENDPOINT from execution variables
+	te, err := getTracetestEndpointFromVars(envManager)
+	if err != nil {
+		outputPkg.PrintLog(fmt.Sprintf("%s [TracetestRunner]: TRACETEST_ENDPOINT variable was not found", ui.IconCross))
+		return result, err
+	}
+
+	// Get TRACETEST_OUTPUT_ENDPOINT from execution variables
+	toe, _ := getTracetestOutputEndpointFromVars(envManager)
```
Comment (vsukhin) [inline contrib/executor/tracetest/pkg/runner/runner.go:69]: shouldn't we log a error there?

File: contrib/executor/tracetest/tracetest-test.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  description: Get a Pokemon
+  trigger:
+    type: http
+    httpRequest:
+      url: http://demo-pokemon-api.demo/pokemon?take=20&skip=0
+      method: GET
+      headers:
+        - key: Content-Type
+          value: application/json
+  # specs:
```
Comment (vsukhin) [inline contrib/executor/tracetest/tracetest-test.yaml:13]: should it be removed?

**Review by vsukhin — APPROVED** [review]
looks good. a few minor comments and need to add to ci/cd flow

File: contrib/executor/tracetest/pkg/runner/runner.go (hunk 1 above)
Comment (vLia) [reply contrib/executor/tracetest/pkg/runner/runner.go]: yes, we should, added it

File: contrib/executor/tracetest/pkg/runner/runner.go [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+	// Configure Tracetest CLI
```
[the remark this answers is not in the document]
Comment (vLia) [reply contrib/executor/tracetest/pkg/runner/runner.go]: removed it

File: contrib/executor/tracetest/tracetest-test.yaml (hunk 2 above)
Comment (vLia) [reply contrib/executor/tracetest/tracetest-test.yaml]: the whole file can be found in the docs, will remove it for now

**Pull request merged**